website/docs/registries/packaging.md

```diff
@@ -0,0 +1,33 @@
+# Packaging your project
```

Comment:
I think `Publishing` is more appropriate, since it's the purpose

Reply:
Addressed in 247cb0390d313743b6832a3ab02a0598a203d224